### PR TITLE
FIX-#3420: update cloud images

### LINKED ISSUE #3420: Cloud examples are outdated
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: `Ubuntu 20.04`
- **Modin version** (`modin.__version__`): 0a642753af2cd2332ac71166bcddc89b0cace9d5
- **Python version**: `3.8.10`
- **Code we can use to reproduce**:

### Describe the problem

The image in use is no longer available from `AWS` provider, so all examples with the cloud do not work.

UPD: during testing, other types of errors were found, which will be discussed below.

### PR DESCRIPTION
**Pull request by anmyachev**
Signed-off-by: Anatoly Myachev <anatoly.myachev@intel.com>

## What do these changes do?

- [ ] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [ ] passes `flake8 modin`
- [ ] passes `black --check modin`
- [ ] signed commit with `git commit -s` 
- [x] Resolves #3420 
- [ ] tests added and passing
- [ ] module layout described at `docs/developer/architecture.rst` is up-to-date

### CODE REVIEW

File: examples/cluster/taxi-runner.py [hunk 1]
```diff
@@ -35,20 +35,25 @@
...
-    data_file = "https://modin-datasets.s3.amazonaws.com/trips_data.csv"
     if USE_OMNISCI:
-        # Workaround for GH#2099
         from modin.experimental.cloud import get_connection
 
-        data_file, remote_data_file = "/tmp/trips_data.csv", data_file
-        get_connection().modules["subprocess"].check_call(
-            ["wget", remote_data_file, "-O", data_file]
+        # We should move omniscripts trigger in remote conext
+        import modin.experimental.engines.omnisci_on_native.frame.omnisci_worker
```
[the remark this answers is not in the document]
Comment (vnlitvinov) [reply examples/cluster/taxi-runner.py:48]: please put the link in the code as a comment

**Review by vnlitvinov — COMMENTED** [review]
Code looks good, but cannot approve until the link is in the code.

File: examples/cluster/taxi-runner.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (anmyachev) [reply examples/cluster/taxi-runner.py]: ok

**Review by vnlitvinov — APPROVED** [review]
LGTM

**Pull request merged**